src/main/groovy/com/athaydes/automaton/SwingUtil.groovy

```diff
@@ -137,12 +137,23 @@ class SwingUtil {
 		}
 		for ( row in rows ) {
 			for ( col in cols ) {
-				if ( action( table.model.getValueAt( row, col ), row, col ) ) return true
+				if ( action( getRenderedTableValue(table, row, col ), row, col ) ) return true
 			}
 		}
 		return false
 	}
 
+	private static Object getRenderedTableValue(JTable table, row, col) {
```

Comment:
Please omit the return type (Object does not give any useful information) and, as `row` and `col` must be `int` for the code to work, add the types there.